src/CNLOADS.CPP

```diff
@@ -4499,9 +4469,9 @@ RC RSYS::rs_SetupASHP()		// set ASHP defaults and derived parameters
 	// inter-default cap95 / cap47
 	//   at least one is present (see rs_CkF)
 	if (!IsSet( RSYS_CAP95))
-		rs_cap95 = rs_Cap95FromCap47ASHP( rs_cap47);
+		ashpCap95FromCap47( rs_cap47, IsSet(RSYS_CAPRAT9547), rs_capRat9547);
 	else if (!IsSet( RSYS_CAP47))
-		rs_cap47 = rs_Cap47FromCap95ASHP( rs_cap95);
+		ashpCap47FromCap95( rs_cap95, IsSet(RSYS_CAPRAT9547), rs_capRat9547);
```

Comment:
Ditto rs_cap47 = ashpCap47FromCap95( ... )

Alarming that tests did not pick this up.